src/scss/layout/_hero.scss

```
@@ -7,11 +7,11 @@
   position: relative;
 
   &::before {
-    @include tools.position(0 -1em 0 -1em);
-
     box-shadow: 0 0 var(--quarter-shim) var(--shadow) inset;
     content: '';
+    inset: 0 -1em 0 -1em;
```

I think we can use the 2-value version for this?

```suggestion
    inset: 0 -1em;
```

---

@mirisuzanne changed this to the 2-value version